encryptedlogging/src/main/kotlin/com/automattic/encryptedlogging/store/OnChanged.kt

```diff
@@ -1,8 +1,8 @@
 package com.automattic.encryptedlogging.store
 
-internal open class OnChanged<T : OnChangedError> {
-    var error: T? = null
-    fun isError(): Boolean {
+public open class OnChanged<T : OnChangedError> {
+    public var error: T? = null
+    public fun isError(): Boolean {
         return error != null
     }
```

Comment:
As this is now part of the public contract, can we remove this to reduce classes exposed to clients? There's no value in this abstraction, it's just a leftover from FluxC.

Reply:
Done: 9f037323f39aa0fc80310fea975c9d683f8a79a9 @wzieba